docs/GettingStarted/configuration/ProductionBestPractice.md

```diff
@@ -71,24 +61,27 @@ metadata:
   namespace: pelorus
 spec:
   exporters:
-    global: {}
-    instances:
-      - app_name: deploytime-exporter
-        exporter_type: deploytime
-      - app_name: failuretime-exporter
-        exporter_type: failure
-      - app_name: committime-exporter
-        exporter_type: committime
-  extra_prometheus_hosts: null
-  openshift_prometheus_basic_auth_pass: changeme
-  openshift_prometheus_htpasswd_auth: 'internal:{SHA}+pvrmeQCmtWmYVOZ57uuITVghrM='
+    [...] # Pelorus exporters configuration options
+
+  # Internal user password
+  openshift_prometheus_basic_auth_pass: mysecretpassword
+  openshift_prometheus_htpasswd_auth: 'internal:{SHA}CM2SM2eJAAllfquBJ1M3m9syHus='
+
+  # Configure Prometheus Retention and PV
   prometheus_retention: 1y
-  prometheus_retention_size: 1GB
+  prometheus_retention_size: 2GB
   prometheus_storage: true
   prometheus_storage_pvc_capacity: 20Gi
   prometheus_storage_pvc_storageclass: gp2
+
+  # Configure Long Term Storage
   thanos_bucket_name: <bucket name here>
-  bucket_access_point: s3.us-east-2.amazonaws.com
+  bucket_access_point: <s3 access point here>
   bucket_access_key: <your access key>
   bucket_secret_access_key: <your secret access key>
 ```
+
+## Deploying Across Multiple Clusters
```

Comment:
I would move this section to pelorus core (at the bottom of it as well)

Reply:
I have moved content of this to the pelorus core and here left the link, because it makes sense to me to inform user about such scenario directly here. Also note that this part is in the Architecture to explain how this works.